src/3d/symbols/qgspointcloud3dsymbol_p.cpp

```diff
@@ -583,6 +585,17 @@ void QgsSingleColorPointCloud3DSymbolHandler::processNode( QgsPointCloudIndex *p
   const QgsCoordinateTransform coordinateTransform = context.coordinateTransform();
   bool alreadyPrintedDebug = false;
 
+  // The request was made on the boundingbox of the real extent
+  // It is necessary to ensure that the features intersect
+  // with extentGeom if the scene rotation is not 0.
+  QgsGeos extentIntersectionGeos( nullptr );
```

Comment:
Looking better! If you look through the code, the typical pattern is to use a unique_ptr for QgsGeos objects to avoid an invalid construction like this.